src/containers/pages/ExportPage/ExportPage.js

```diff
@@ -0,0 +1,186 @@
+  static propTypes = {
+    articles: PropTypes.instanceOf(Immutable.List),
+    loadEntities: PropTypes.func,
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.bind();
+    this.state = { selectedRows: [] };
+  }
+
+  getSelectedRows(articles, selectedRows) {
+    if (articles.isEmpty() ||
+        selectedRows === 'none' ||
+        (Array.isArray(selectedRows) && selectedRows.length === ZERO)) {
+      return Immutable.fromJS([]);
+    }
+
+    if (selectedRows === 'all') {
+      return articles;
+    }
+
+    return articles.filter((x, i) => selectedRows.includes(i));
+  }
+
+  bind() {
+    this.handleOnExport = this.handleOnExport.bind(this);
+    this.handleOnRowSelection = this.handleOnRowSelection.bind(this);
+    this.handleOnSave = this.handleOnSave.bind(this);
+  }
+
+  handleOnExport() {
+    this.props.loadEntities({ href: '/bookmarks', type: ARTICLES_VIEW_STATE, schema: arrayOf(articleSchema) });
+  }
+
+  handleOnSave() {
+    const tempLink = document.createElement('a');
+    const newArticles = this.getSelectedRows(this.props.articles, this.state.selectedRows);
+    const content = encodeURIComponent(JSON.stringify(newArticles));
+
+    if (newArticles.isEmpty()) {
+      return;
+    }
+
+    tempLink.setAttribute('href', `data:text/plain;charset=utf-8,${content}`);
+    tempLink.setAttribute('download', 'heutagogy.json');
+
+    if (document.createEvent) {
+      const event = document.createEvent('MouseEvents');
+
+      event.initEvent('click', true, true);
+      tempLink.dispatchEvent(event);
+    } else {
+      tempLink.click();
+    }
+
+    this.setState({ selectedRows: [] });
+  }
+
+  handleOnRowSelection(selectedRows) {
+    this.setState({ selectedRows });
+  }
+
+  renderTable() {
+    return (
+      <Table
+        multiSelectable
+        onRowSelection={this.handleOnRowSelection}
+      >
+        <TableHeader>
+          <TableRow>
+            <TableHeaderColumn>{'Link'}</TableHeaderColumn>
+            <TableHeaderColumn>{'Title'}</TableHeaderColumn>
+            <TableHeaderColumn>{'Saved'}</TableHeaderColumn>
+            <TableHeaderColumn>{'Read'}</TableHeaderColumn>
+          </TableRow>
+        </TableHeader>
+        <TableBody
+          deselectOnClickaway={false}
+          showRowHover
+        >
+          {this.props.articles.map((item, i) => {
+            return (
+              <TableRow
+                key={i}
+                selected={this.state.selectedRows.indexOf(i) !== MINUS_ONE}
+              >
+                <TableRowColumn>
+                  <a
+                    href={item.get('url')}
+                    target="_blank"
+                  >
+                    {item.get('url')}
+                  </a>
+                </TableRowColumn>
+                <TableRowColumn>{item.get('title')}</TableRowColumn>
+                <TableRowColumn>{moment.unix(item.get('timestamp') / THOUSAND).format('MM/DD/YYYY hh:mm')}</TableRowColumn>
```

Comment:
I have no idea (ping @Faradey27 :wink: ) − I grabbed it from [ArticlesPage.js#L96](https://github.com/heutagogy/heutagogy-frontend/blob/master/src/containers/pages/ArticlesPage/ArticlesPage.js#L96)